sockeye/model.py

```diff
@@ -278,7 +282,7 @@ def _embed_and_encode(self,
     def decode_step(self,
                     step_input: pt.Tensor,
                     states: List[pt.Tensor],
-                    vocab_slice_ids: Optional[pt.Tensor] = None) -> Tuple[pt.Tensor, List[pt.Tensor], List[pt.Tensor]]:
+                    vocab_slice_ids: Optional[pt.Tensor] = None) -> Tuple[pt.Tensor, pt.Tensor, List[pt.Tensor], List[pt.Tensor]]:
```

Comment:
docstring needs update

Reply:
@tdomhan seems like you changed this line -- any idea what docstring update needs to be done?

Reply:
```
:return: logits, list of new model states, other target factor logits.
```

```
:return: logits, KNN output if present otherwise None, list of new model states, other target factor logits.
```

Reply:
Formatting: Break line to fit 120-character margin.